pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/BrokerServerBuilder.java

```diff
@@ -244,8 +244,12 @@ public void start() throws Exception {
 
   public void stop() throws Exception {
     LOGGER.info("Shutting down Network !!");
-
     _state.set(State.SHUTTING_DOWN);
+    try {
+      Thread.sleep(10000);
```

Comment:
Mostly for both health check and externalView to be in offline status.
I've observed many cases when I'm bouncing broker, the client side with see a short period that query response is empty.
I tried to setup both haproxy pinging health check  and also register an externalView listener. Since we are stopping too fast, either of this approach will treat this broker is still alive, but actually getting empty results.

The clean way here is to set idealstates of this broker to offline meanwhile still serving queries. Then  wait for externalView's reflection to offline then shutdown broker.

Also a configurable fixed timeout is necessary if I'm using haproxy to do health check.